### ISSUE TITLE
Using versions 3.0 and above causes an error

### ISSUE THREAD

**Issue by Paxnar** [JuanBindez/pytubefix#56]
**Describe the bug**
I tried version 3.0 and some versions above that because the videos I tried to download on v2.0 just said "Vide not available". Now when I try to run any code, I get an error in the import line.

```
Traceback (most recent call last):
File "C:/Users/USER/PycharmProjects/testing/youturb.py", line 1, in <module>
    import pytubefix as pytube
  File "C:\Users\USER\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pytubefix\__init__.py", line 13, in <module>
    from pytubefix.streams import Stream
  File "C:\Users\USER\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pytubefix\streams.py", line 19, in <module>
    from pytubefix import extract, request
  File "C:\Users\USER\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pytubefix\extract.py", line 92, in <module>
    def playability_status(watch_html: str) -> tuple[str, str]:
TypeError: 'type' object is not subscriptable
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Python Version 3.8
 - Pytubefix Version 3.0, also tried 3.0rc1 and 3.1rc1

**Comment by JuanBindez** [comment]
show the code being tested

**Comment by Paxnar** [comment]
> show the code being tested

this error occurs on importing pytubefix, so it happens with any code

```
import pytubefix
```

**Comment by JuanBindez** [comment]
This error is not occurring here

**Comment by JuanBindez** [comment]
try

```python

from pytubefix import YouTube

```

**Comment by Paxnar** [comment]
> try
> 
> ```python
> from pytubefix import YouTube
> ```

same error

**Comment by Paxnar** [comment]
my only guess is that tuple[str, str] is a syntax that isn't implemented in Python 3.8, which I am using
